### PR TITLE
feat: added Wishlist feature & fixed links in other pages

### PR DESCRIPTION
**Pull request by zabihshaik**
Added Wishlist Management Feature Page and imported it's CSS file in styles.css

Wishlist Page looks like this:

Wishlist Page

### CODE REVIEW

**Review by yashpatel1797 — APPROVED** [review]
looks good to merge

**Review by agrawalshanu708 — APPROVED** [review]
looks good to merge

File: styles/product.css [hunk 1]
```diff
@@ -1,11 +1,7 @@
 .product-sidebar {
     position: sticky;
     top: 0;
-    bottom:0;
-}
-
-.advertisement {
-    margin-top: 1rem;
+    bottom: 0;
```
Comment (anandkumarcoder) [inline styles/product.css:4]: remove white spaces

File: styles/product.css [hunk 2]
```diff
@@ -27,6 +23,15 @@
     width: 18rem;
 }
 
+.card-heading {
+    font-weight: 600;
+}
+
```
Comment (anandkumarcoder) [inline styles/product.css:29]: same white spaces again

File: styles/wishlist.css [hunk 3]
```diff
@@ -0,0 +1,16 @@
+.wishlist-container {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-wrap: wrap;
+    gap: 1rem;
+    margin: 2rem auto;
+}
+
```
Comment (anandkumarcoder) [inline styles/wishlist.css:9]: use prettier

**Review by anandkumarcoder — COMMENTED** [review]
just remove white space and use orettier for formatting

File: styles/product.css (hunk 1 above)
Comment (zabihshaik) [reply styles/product.css:4]: That was actually deleted.

File: styles/product.css (hunk 2 above)
Comment (zabihshaik) [reply styles/product.css:29]: Yeah. Will do it. Thanks.

**Comment by zabihshaik** [comment]
Fixed minor changes as per PR Review

**Pull request merged**